evap/static/scss/evap.scss

```diff
@@ -146,3 +146,48 @@ td.icon {
 @media print {
     @import "print";
 }
+
+.notebook-animation {
+    transition: width .0s !important;
+}
+
+@media screen and (orientation: landscape) {
+    .notebook-margin {
+        max-width: 75vw;
+        margin-left: 25vw;
+    }
+
+    .notebook-container {
+        width: 25vw;
+        padding-left: $spacer;
+        padding-bottom: $spacer;
+        padding-top: $spacer;
+    }
+
+    .notebook-card {
+        // substracting 3 * Margins 2x card 1x breadcrumb
+        height: calc((100vh - #{$navbar-brand-height}) - (#{$footer-height} + ((#{$spacer} * 1.5) * 3))) !important;
```

Comment:
1. Why is the spacer multiplied by 1.5?
2. Why do we need the `!important`? (=> what other rule would override the height without `!important`?)

Reply:
this is multiplied by 1.5 to match the margins. Have a look at the bootsrap docs. I am going to search for a more elegant solution.